alfasim_sdk/types.py

```diff
@@ -44,24 +46,27 @@ class String(BaseField):
 
 
     """
-    value: str = attrib(validator=instance_of(str))
+    value: str = attrib(validator=[instance_of(str), check_string_is_not_empty])
 
 
 @attr.s(kw_only=True)
 class Enum(BaseField):
-    value: List[str] = attrib()
+    values: List[str] = attrib()
     initial: str = attrib(validator=optional(instance_of(str)), default=None)
 
-    @value.validator
-    def check(self, attr, values):
-        if not isinstance(values, list):
-            raise TypeError(f"{attr.name} must be a list, got a {type(values)}.")
+    @values.validator
+    def check(self, attr, user_values):
+        if not isinstance(user_values, list):
+            raise TypeError(f"{attr.name} must be a list, got a {type(user_values)}.")
 
-        if not all(isinstance(value, str) for value in values):
+        if not all(isinstance(value, str) for value in user_values):
             raise TypeError(f"{attr.name} must be a list of string.")
```

Comment:
"list of strings"

Also include the offending item on the message. You might want to merge this check with th e`check_string_is_not_empty` below

Reply:
What do you think?
```python
    with pytest.raises(TypeError, match="values must be a list of string, the item '1' is a 'int'"):
        Enum(values=[1], caption='caption')

Reply:
Just a typo: "list of strings" ;)